### PR TITLE
[Merged by Bors] - chore(Order): move more defs to Defs

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

File: Mathlib/Order/Minimal.lean [hunk 1]
```diff
@@ -52,24 +52,6 @@ section LE
```
Comment (sgouezel) [inline Mathlib/Order/Minimal.lean:52]: The file-level docstring is not correct any more, as it says this file defines minimality. Can you update it?

**Review by sgouezel — COMMENTED** [review]
bors d+

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**